src/common/area-tooltip.component.ts

```diff
@@ -195,4 +197,29 @@ export class AreaTooltip implements OnChanges {
     this.anchorOpacity[index] = 0;
   }
 
+  /** This is a more flexible version of the original template "{{tooltipItem.series}}: {{tooltipItem.value.toLocaleString()}}" */
+  getToolTipText(tooltipItem: any): string {
```

Comment:
The tooltips miss the ending bracket here:

![image](https://cloud.githubusercontent.com/assets/8687/21855072/6000d802-d81d-11e6-951b-57e463f00cb2.png)